nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+
+const array = process.argv.slice(2).map(num => Number(num));
+if (array.length === 0) {
+    console.log('No arguments are provided')
+}
+
```

Comment:
Normally else and elseif should start right where if statement closes. Like
https://stackoverflow.com/a/4005623/4261654